### PR TITLE
Schedule Activity Page Improvements

### PR DESCRIPTION
**Pull request by mbouadeus**
Add loading indicator when the user submits the schedule activity form: https://screenshot.googleplex.com/5kXB3RNKWtV9ukU.png
Added error message for create event request failers: https://screenshot.googleplex.com/BTtGodVcHkRR6c9.png
Fix authentication error
Fixed bug where videos displayed articles selected
Made user automatically part of the guest list so they are listed as a guest on the calendar event.

### CODE REVIEW

**Review by trinitydonohugh — APPROVED** [review]
LGTM.

File: src/main/java/com/google/sps/data/CalendarUtility.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public static Calendar loadCalendarClient(String token) throws IOException {
     // Create token response object
     TokenResponse tokenObj = new TokenResponse();
     tokenObj.setAccessToken(token);
-
+
```
Comment (pengxianzhe) [inline src/main/java/com/google/sps/data/CalendarUtility.java:79]: nit: it looks like you added two spaces for this empty line. Please remove.

File: src/main/java/com/google/sps/servlets/CreateEventServlet.java [hunk 2]
```diff
@@ -38,8 +39,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Map<String, String> eventInfo = getParameters(request);
 
     EventUtility eventUtility = new EventUtility();
-    ActivityEvent event = eventUtility.getActivityEvent(eventInfo, 
-      KeyFactory.stringToKey(eventInfo.get("activityKey")));
+    Key activityKey = null;
+    try {
+      activityKey = KeyFactory.stringToKey(eventInfo.get("activityKey"));
+    } catch (IllegalArgumentException ex) {
+      System.out.println(eventInfo.get("activityKey"));
```
Comment (pengxianzhe) [inline src/main/java/com/google/sps/servlets/CreateEventServlet.java:46]: Just curious: is this test code or error logging?

**Review by pengxianzhe — APPROVED** [review]
LGTM with a couple of comments

**Pull request merged**